runner/ansible/callback_plugins/trento.py

```diff
@@ -152,21 +163,25 @@ def v2_runner_on_ok(self, result):
         test_result = result._task_fields["args"]["test_result"]
         for group in task_vars["group_names"]:
             self.results.set_host_state(group, host, True)
+            if self.results.result_exist(group, task_vars[CHECK_ID], host):
+                continue
             self.results.add_result(group, task_vars[CHECK_ID], host, test_result)
 
-    def v2_runner_on_failed(self, result):
+    def v2_runner_on_failed(self, result, ignore_errors):
```

Comment:
It looks like the `ignore_errors` is never used.

Reply:
Yes. This argument is coming from the placeholder function. Think about it as a golang interface.
We are just implementing the method, that it is defined somewhere else.
In our case, we don't use the argument, but It must go there